api/plugin_support.h

```diff
@@ -188,10 +254,32 @@ inst_set mambo_get_inst_type(mambo_context *ctx);
 int mambo_get_fragment_id(mambo_context *ctx);
 cc_type mambo_get_fragment_type(mambo_context *ctx);
 int mambo_get_inst_len(mambo_context *ctx);
+
+/**
+ * @brief Returns the address of the currently scanned untranslated instruction.
+ *
+ * Returns the untranslated address of the application's instructions that is
+ * currently being scanned by mambo using the plugin's context.
```

Comment:
Plugin context doesn't scan instructions. Scanner (shared between all plugins) does. Context just holds the state of MAMBO exposing information from the scanner.